twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -165,15 +171,16 @@ def _generate_jvm_code(ctx, label, compile_thrifts, include_thrifts, jar_output,
         worker_arg_pad + _colon_paths(ps)
         for ps in [compile_thrifts, include_thrifts, [], []]
     ])
+
+    compiler_args = getattr(ctx.rule.attr, "compiler_args", [])
+    lang_flag = ["--language", language]
+    flags = compiler_args + lang_flag
+
     worker_content = "{output}\n{paths}\n{flags}".format(
         output = jar_output.path,
         paths = path_content,
-        flags = worker_arg_pad + ":".join([
-            # always add finagle option which is a no-op if there are no services
-            # we could put "include_services" on thrift_info, if needed
-            "--with-finagle",
-            "--language={}".format(language),
-        ]),
+        # we could put "include_services" on thrift_info, if needed
```

Comment:
I think the top half of this comment is in the compiler scala code, is this a leftover?